src/routes/github/configuration/github-configuration-router.test.ts

```diff
@@ -125,22 +125,6 @@ describe("Github Configuration", () => {
 
 			githubUserTokenNock(sub.gitHubInstallationId);
 
-			githubNock
```

Comment:
Why have we simply removed these tests?

Reply:
Reverted back and just removed `twice`. These calls are not made twice anymore, cause of the removal of the duplicated lines in `github-configuration-get`